### PR TITLE
T hq tagger in dev legacy run ii

### PR DESCRIPTION
**Pull request by prafullasaha**

### CODE REVIEW

File: Systematics/test/workspaceStd.py [hunk 1]
```diff
@@ -411,6 +413,13 @@
 #    else:
     fc.addObservables(process, process.tagsDumper, customize.processId )
 
+import flashgg.Taggers.THQLeptonicTagVariables as var
```
Comment (simonepigazzini) [inline Systematics/test/workspaceStd.py]: Please put this within a if block that is accessed only for tHq debug.

File: Systematics/test/workspaceStd_thq.py [hunk 2]
```diff
@@ -0,0 +1,736 @@
+#!/usr/bin/env cmsRun
```
Comment (simonepigazzini) [inline Systematics/test/workspaceStd_thq.py]: Is this cfg used for debugging purposes? If yes it might be worth to move it to Validation/test

Comment (simonepigazzini) [reply Systematics/test/workspaceStd_thq.py]: also it looks mostly as a copy and paste from wrokspaceStd.py, so maybe you can have the features implemented there with a debug tHq switch

File: Taggers/plugins/THQLeptonicTagProducer.cc_cleaned [hunk 3]
```diff
@@ -0,0 +1,1693 @@
+﻿#include "FWCore/Framework/interface/EDProducer.h"
```
Comment (simonepigazzini) [inline Taggers/plugins/THQLeptonicTagProducer.cc_cleaned]: It looks like this file was commited by mistake.

**Comment by prafullasaha** [comment]
Hi, I removed THQLeptonicTagProducer.cc_cleaned and workspaceStd_thq.py file as I commited those two file by mistake and in workspaceStd.py I have put tHq variable part in a if block. Thank You

**Comment by simonepigazzini** [comment]
Hello,
can you please also add this snippet at the end of each new c++ file:
// Local Variables:
// mode:c++
// indent-tabs-mode:nil
// tab-width:4
// c-basic-offset:4
// End:
// vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4

Thank you

**Comment by prafullasaha** [comment]
Hi,
 I have added snippet at the end of each new c++ file

Thank You

**Comment by simonepigazzini** [comment]
Merged with PR #1133

**Pull request closed without merging**